syntax = "proto3";

package myservice;

service MyService {
    // Sends a greeting
    rpc MyMethod (MyRequest) returns (MyResponse) {}
}

message MyRequest {
    string name = 1;
}

message MyResponse {
    string message = 1;
}